src/views/tables/PositionsTable.tsx

```diff
@@ -453,25 +355,25 @@ const getPositionsTableColumnDef = ({
         hideOnBreakpoint: MediaQueryKeys.isTablet,
         align: 'center',
         renderCell: ({
-          id,
+          market,
           assetId,
           tickSizeDecimals,
           liquidationPrice,
           side,
-          size,
+          signedSize,
           stopLossOrders,
           takeProfitOrders,
         }) => {
           return (
             <PositionsTriggersCell
-              marketId={id}
+              marketId={market}
```

Comment:
this marketId is not the displayableId. When users go to share pnl they may have long form tickers